mwdb/model/user.py

```diff
@@ -63,6 +63,10 @@ class User(db.Model):
         "Object", secondary=favorites, back_populates="followers", lazy="joined"
     )
 
+    commented_objects = db.relationship(
+        "Object", secondary="comment", back_populates="comment_authors", lazy="selectin"
```

Comment:
I would left that backref without eager loading, so we don't load all commented objects every time we're accessing the user.
```suggestion
        "Object", secondary="comment", back_populates="comment_authors"
```

Reply:
Corrected.